components/grid-info.vue

```diff
@@ -0,0 +1,211 @@
+<template>
+  <div class="block">
+    <div class="grid">
+      <div class="block">
+        <div class="grid__block">
+          <div class="title">Open Source</div>
+          <div class="text">Исходный код открыт для исследования и обслуживания</div>
+        </div>
+        <div class="grid__block">
+          <div class="title">100 000 000</div>
+          <div class="text">Событий обрабатывается<br>каждый день</div>
+        </div>
+      </div>
+      <div class="grid__block">
+        <div class="title">Проверено продуктами</div>
+        <div class="text">Hawk используется рядом хорошо известных продуктов.<br>И еще большим количеством менее известных.</div>
+        <div class="products">
+          <VcIcon class="svg, products__box1"/>
+          <DtfIcon class="svg, products__box2"/>
+          <EditorIcon class="svg, products__box3"/>
+          <HeykaIcon class="svg, products__box4"/>
+          <TjIcon class="svg, products__box5"/>
+        </div>
+      </div>
+    </div>
+    <div class="grid__block">
+      <div class="stack">
+        <div class="cell">
+          <img src='~/assets/svg/ru/vue.svg' class="svg"/>
+        </div>
+        <div class="cell">
+          <img src='~/assets/svg/ru/vite.svg' class="svg"/>
+        </div>
+        <div class="cell">
+          <img src='~/assets/svg/ru/webpack.svg' class="svg"/>
+        </div>
+        <div class="cell">
+          <img src='~/assets/svg/ru/js.svg' class="svg"/>
+        </div>
+        <div class="cell">
+          <img src='~/assets/svg/ru/node.svg' class="svg"/>
+        </div>
+        <div class="cell">
+          <img src='~/assets/svg/ru/kotlin.svg' class="svg"/>
+        </div>
+        <div class="cell">
+          <img src='~/assets/svg/ru/python.svg' class="svg"/>
+        </div>
+        <div class="cell">
+          <img src='~/assets/svg/ru/go.svg' class="svg"/>
+        </div>
+        <div class="cell">
+          <img src='~/assets/svg/ru/php.svg' class="svg"/>
+        </div>
+      </div>
+      <div class="title">Поддерживает<br/>ваши технологии</div>
+      <div class="text">Хоук имеет интеграции с популярными<br/>технологиями и фреймворками. </div>
```

Comment:
margin between title and description is too big

![image](https://github.com/user-attachments/assets/d2fb2fed-c7a9-42e0-8219-5cc458dd308a)